Documentation/github-connector.md

```diff
@@ -29,11 +28,26 @@ connectors:
     clientID: $GITHUB_CLIENT_ID
     clientSecret: $GITHUB_CLIENT_SECRET
     redirectURI: http://127.0.0.1:5556/dex/callback
-    # Optional organization to pull teams from, communicate through the
+    # Optional organizations and teams, communicated through the
     # "groups" scope.
     #
     # NOTE: This is an EXPERIMENTAL config option and will likely change.
-    org: my-oranization
+    #
+    # Legacy 'org' field. 'org' and 'orgs' cannot be used simultaneously. Use
+    # this field to restrict access to the specified GitHub org.
+    # org: my-organization
+    #
+    # New 'orgs' field. 'orgs' supports organizations with or without teams.
```

Comment:
Few nits.

* This field is 'new' in the release notes, for docs it just is.
* "restrict access" is probably the wrong phrase.
* Let's just remove the `NOTE: ...` line and `Legacy 'org' field...` block.

Maybe something like this:

```yaml
# Query the following organizations for group information if the "groups"
# scope is provided. Group claims are formatted as "(org):(team)". For
# example if a user is part of the "engineering" team of the "coreos" org,
# the group claim would include "coreos:engineering"
#
# A user MUST be a member of at least one of the following orgs to
# authenticate with dex.
orgs:
- name: my-org
  # Include all teams as claims.
- name: my-org-with-teams
  # A white list of teams. Only include group claims for these teams.
  teams:
  - read-team
  - blue-team
```